exporter/collector/config.go

```diff
@@ -59,6 +59,20 @@ type MetricConfig struct {
 	// Buffer size for the channel which asynchronously calls CreateMetricDescriptor. Default
 	// is 10.
 	CreateMetricDescriptorBufferSize int `mapstructure:"create_metric_descriptor_buffer_size"`
+
+	// If true, the exporter will copy OTel's service.name, service.namespace, and
+	// service.instance.id resource attributes into the GCM timeseries metric labels. This
+	// option is recommended to avoid writing duplicate timeseries against the same monitored
+	// resource. Default is true.
+	IncludeServiceResourceAttributes bool `mapstructure:"include_service_resource_attributes"`
```

Comment:
Can we unify the naming of this and instrumentation library labels?  E.g.
`instrumentation_library_labels` and `service_resource_labels`
or
`include_instrumentation_library_attributes` and `include_service_resource_attributes`

Reply:
Do you have a preference between those two?

Reply:
I'd prefer `instrumentation_library_labels` and `service_resource_labels`